docs/source/tutorials/open_images_evaluation.ipynb

```diff
@@ -0,0 +1,444 @@
+    "# MODEL_HANDLE=\"https://tfhub.dev/google/openimages_v4/ssd/mobilenet_v2/1\"\n",
+    "\n",
+    "python open_images_eval/scripts/inference.py \\\n",
+    "    --output_dir ${OUTPUT_DIR} \\\n",
+    "    --output_format tf_object_detection_api \\\n",
+    "    ${IMAGES_DIR} ${MODEL_HANDLE}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Load the data into FiftyOne\n",
+    "\n",
+    "We are going to use the [fiftyone](https://github.com/voxel51/fiftyone) package\n",
+    "for visualizing the data."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Installing FiftyOne\n",
+    "\n",
+    "See [FiftyOne Installation](https://voxel51.com/docs/fiftyone/getting_started/install.html)\n",
+    "for up-to-date instructions."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create a persistent FiftyOne dataset\n",
+    "\n",
+    "The following script loads the data into a FiftyOne [Dataset](https://voxel51.com/docs/fiftyone/api/fiftyone.core.dataset.html#fiftyone.core.dataset.Dataset).\n",
+    "This process copies all labels and metadata to a non-relational database for rapid access and powerful querying, but only paths to the images are stored\n",
+    "in the database, not copies of the images themselves!\n",
+    "\n",
+    "The dataset is set to [persistent](https://voxel51.com/docs/fiftyone/user_guide/using_datasets.html#dataset-persistence)\n",
+    "so that it remains in the database and can be loaded in a new python process."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "DATASET_NAME=\"open-images-v4-test\"\n",
+    "IMAGES_DIR=/PATH/TO/IMAGES\n",
+    "BOUNDING_BOXES_EXPANDED=/PATH/TO/test-annotations-bbox_expanded.csv\n",
+    "IMAGE_LABELS_EXPANDED=/PATH/TO/test-annotations-human-imagelabels-boxable_expanded.csv\n",
+    "PREDICTIONS_PATH=/PATH/TO/PREDICTIONS.csv\n",
+    "CLASS_DESCRIPTIONS=/PATH/TO/class-descriptions-boxable.csv\n",
+    "\n",
+    "python open_images_eval/scripts/load_data.py \\\n",
+    "    --bounding_boxes_path ${BOUNDING_BOXES_EXPANDED} \\\n",
+    "    --image_labels_path ${IMAGE_LABELS_EXPANDED} \\\n",
+    "    --predictions_path ${PREDICTIONS_PATH} \\\n",
+    "    --prediction_field_name \"faster_rcnn\" \\\n",
+    "    --class_descriptions_path ${CLASS_DESCRIPTIONS} \\\n",
+    "    --load_images_with_preds \\\n",
+    "    --max_num_images 1000 \\\n",
+    "    ${DATASET_NAME} ${IMAGES_DIR}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### (optional) Visualize the data\n",
+    "\n",
+    "Now that we have a Fiftyone `Dataset`, we can optionally visualize the data before evaluating.\n",
+    "Open up a `python` or `ipython` terminal and run the following:\n",
+    "\n",
+    "```python\n",
+    "import fiftyone as fo\n",
```

Comment:
I don't see any reason why this shouldn't be a cell I can run.

Reply:
Because you are using a bash kernel, not a python kernel

Reply:
Right, that makes a lot of sense.... This makes it feel really weird though, having a python notebook that can only run bash code blocks and you need to copy and paste the python code into a terminal. I feel like this tutorial could be streamlined a lot by using a python kernel, still being able to run the installs through `!pip install ...` and then just having to copy the 3 scripts (assuming you don't have to run inference anymore)

Reply:
okay I switched it to `%%bash` blocks